packages/remotemetrics/events.go

```diff
@@ -79,8 +82,9 @@ type MessageScheduledMetrics struct {
 	GradeOfFinality          uint8     `json:"gradeOfFinality" bson:"GradeOfFinality"`
 	DeltaGradeOfFinalityTime int64     `json:"deltaGradeOfFinalityTime" bson:"deltaGradeOfFinalityTime"`
 	DeltaSolid               int64     `json:"deltaSolid,omitempty" bson:"deltaSolid"`
-	DeltaScheduled           int64     `json:"deltaScheduled" bson:"deltaScheduled"`
-	DeltaBooked              int64     `json:"deltaBooked" bson:"deltaBooked"`
+	// scheduledTimestamp - IssuedTimestamp in nanoseconds
+	DeltaScheduled int64 `json:"deltaScheduled" bson:"deltaScheduled"`
+	DeltaBooked    int64 `json:"deltaBooked" bson:"deltaBooked"`
 	// scheduledTimestamp - ReceivedTimestamp in nanoseconds
 	ProcessingTime int64 `json:"processingTime" bson:"processingTime"`
```

Comment:
I think `processingTime` and `deltaScheduled` have confusing names currently

you have recommendations?

Reply:
`DeltaIssuedScheduled`
`DeltaIssuedBooked`
etc...

`DeltaReceivedScheduled` - that'd be the processing time
`DeltaReceivedBooked`
etc.

that way we could encode what exactly a given field means and it would not be conflicting for deltas from issuance time and receive time.